### DISCUSSION TITLE
Android: :ListView render very slow

### DISCUSSION THREAD

**Discussion by pitush** [unoplatform/uno#10501] [Q&A]
It seems ListView render very slow on android, I'm using <ScrollViewer> <mux:ItemsRepeater /> </ScrollViewer>
It improves performance a lot, still not satisfied as i did have to enlarge items to not exceed 3 items. Any Ideas how to improve?

**Comment by jeromelaban** [comment]
Thanks for the report. Could you provide a sample repro of what you're trying to display?
